config.yaml.example

```diff
@@ -0,0 +1,19 @@
+# customChecks defines custom checks.
+customChecks:
+- name: "required-label-app"
+  template: "required-label"
+  params:
+    key: "app"
+checks:
+  # if doNotAutoAddDefaults is true, default checks are not automatically added.
+  # Otherwise, they are.
```

Comment:
Remove, this is pretty obvious based on the previous line

Reply:
Also, I think doNotAutoAddDefaults is backwards, should be autoAddDefaults that defaults to true

Reply:
it's also just a bit too verbose